considerations.md

```diff
@@ -5,7 +5,7 @@ Instead they MUST ignore unknown properties.
 
 # Canonicalization
 
-OCI Images [are](descriptor.md) [content-addressable](image-layout.md).
+OCI Images are content-addressible. See [Descriptors](descriptor.md) and [Image Layout](image-layout.md) for more information.
```

Comment:
Well, the refs (now `index.json`) are not content addressable.  But @jonboulle's proposal looks good to me if we replace:

> All of the constituent components of the OCI Image specification

with:

> OCI Images

That just assumes that no fundamental part of the image become mutable reefs only, which hopefully remains true going forward ;).